src/main/java/io/sphere/sdk/meta/ReleaseNotes.java

```diff
@@ -177,7 +177,7 @@ In Scala (with add-ons) it will look like <pre><code>val query = ProductProjecti
  <li class=new-in-release>Added the nested attributes: {@link io.sphere.sdk.attributes.AttributeAccess#ofNested()} + {@link io.sphere.sdk.attributes.AttributeAccess#ofNestedSet()}.</li>
  <li class=new-in-release>The error JSON body from SPHERE.IO responses can be directly extracted as JSON with {@link io.sphere.sdk.client.SphereServiceException#getJsonBody()}.</li>
  <li class=new-in-release>{@link io.sphere.sdk.http.HttpResponse} also contains {@link io.sphere.sdk.http.HttpHeaders}.</li>
- <li class=new-in-release>Experimental search predicate model {@link io.sphere.sdk.products.search.ProductProjectionSearch#model()}. See also {@link io.sphere.sdk.meta.ProductSearchDocumentation}.</li>
```

Comment:
Normally release notes of the past should not be changed, since every new change should be in new release notes.
But I also had the case that old links did not work so I changed `{@link }` to `{@code }`.